### ISSUE TITLE
Add autocomplete to forms

### ISSUE THREAD

**Issue by aerni** [statamic/ideas#529]
Add autocomplete attributes to the form builder: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

This could be implemented the same way you already do with the input type.

**Comment by duncanmcclean** [comment]
The 'input type' config option is not special for form blueprints, where as I think this would probably need to be? (unless there's a case for it...)

**Comment by aerni** [comment]
Yes, `autocomplete` only makes sense for forms. What's the case for `input_type` on a regular blueprint? I can't think of one either.

**Comment by duncanmcclean** [comment]
> Yes, `autocomplete` only makes sense for forms. What's the case for `input_type` on a regular blueprint? I can't think of one either.

I know I've used it before in the case of a URL, where it will warn the user if it isn't formatted correctly (eg. they type something that's not a URL)
